regolith/schemas.py

```diff
@@ -1164,6 +1175,21 @@
             "required": False,
             "type": "string",
         },
+        'updated': {
+            "description": "date and time when the entry was created/updated",
```

Comment:
this is the "most recently updated", not the created date

Reply:
done!